### PR TITLE
Move message confirmation from conversation start to the bulk message send action.

### PR DESCRIPTION
**Pull request by jerith**

### CODE REVIEW

File: go/conversation/view_definition.py [hunk 1]
```diff
@@ -329,20 +305,48 @@ def post(self, request, conversation):
...
         self.action.perform_action(action_data)
-        return self.action_done(conversation)
+        return self._action_done(request, conversation)
 
-    def action_done(self, conversation):
+    def _action_done(self, request, conversation):
         next_view = self.get_next_view(conversation)
         if self.action.redirect_to is not None:
             next_view = self.action.redirect_to
+        messages.info(request, 'Action performed succesfully!')
```
Comment (hodgestar) [inline go/conversation/view_definition.py]: Maybe we should include the action name or something in this message?

Comment (jerith) [inline go/conversation/view_definition.py]: Yes. I'll do that now.

**Comment by hodgestar** [comment]
:+1: once the action name is in.

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**